Add test workflow

nop33 opened the pull request.

polarker commented on src/Starter-Guide---How-to-Launch-your-node.md:
Very sweet footnote ❤️

tdroxler commented on src/Starter-Guide---How-to-Launch-your-node.md:
can you fix the `two`? now there's only one variable (we use to have 2)

polarker reviewed (approved):
LGTM

tdroxler commented on src/Starter-Guide---How-to-Launch-your-node.md:
nice, I never did a footnote in md

tdroxler reviewed (approved):
one comment, otherwise LGTM

nop33 commented:
@polarker @tdroxler thanks for your feedback!

I've added the suggestions and moreover I:
1. added a title to the document and changed the hierarchy of the titles of the document's sections from H1 to H2
2. highlighted the explorer link by adding it inside a blockquote

Latest preview:

![image](https://user-images.githubusercontent.com/1579899/141794512-98a93968-ff4d-4fa2-bc80-fabe26dda16f.png)

nop33 commented on src/Starter-Guide---How-to-Launch-your-node.md:
done

polarker commented:
Feel free to merge now, thanks !

nop33 committed:
Add test workflow
